collections.json

```diff
@@ -2001,5 +2001,52 @@
         ]
       }
     ]
+  },
+ {
+    "project": "Vanity Trophies",
+    "collections": [
+      {
+        "logoUrl": "https://raw.githubusercontent.com/AlexDochioiu/nft_craze_gallery_data/main/logos/VanityTrophies.jpeg",
+        "collectionName": "Vanity Trophies",
+        "policies": [
+        "b76f09d72a322783f0187076e620c7aafefd78ffba9187171227db90",
+        "727df24a0af4d9c065d9e8332b4e8e88a44537458310424004899362",
+        "16f43afef2b0b829f79112e0015003b3e25795a4de2f2380ce84c178",
+        "1e06ff6dd01ef2bad53804b38fdcbae219b2522df8ac79f1542a3c8b",
+        "69c0b443c150b0abb872c37630dc5be105a160a78f04453472856be7",
+        "bb4c677c1e33eb4847b33df31d26ab41d4179f9932aea5d21d5cf838",
+        "dc7732e36fd189e4a751fa120a8bf49723e7b3d7b5ef1016e0709566",
+        "9069c3a2bf0e2904f0f9f36cdf415f1608bab46c9b1d304abae844a1",
+        "36423e73d03dbfaf9fcd13f9207476d77c7e637fdf4aa65bc8f8259f",
+        "047b8476ffebc4c8881034aca64409e67ea4ada5a66832233d57fe14",
+        "3234b164d3b25504c7d50a01ea7cb9e50a548e1b6e3b55c579cbaec7",
+        "f691ed90fa3e4fe7c1920c7f0d9638bdb4c07016a5b0e23bd1a02861",
+        "bd03f0e3cef5afe3a44eb58f0668b29f9f4451015fc1af1b41d614c4",
+        "813129e24287c2a63ce9028c26df7c144d770b81937d01772ec2f6f9",
+        "1de1c85018d1ae8b3e1e35019fa98c7a57a16a0dde04f9a1a5b95884",
+        "2005a8f42e57be7d8ac3c916d298225208ff39af020b28e9d671f3ef",
+        "5c08ff27f5defa4f0895d5aab8cb7d8c11e68793e51f39936ec3a316",
+        "abe8d2d78042daa5bce6ae433ed7f9a75137aa2d1f4fbcecaee3b443",
+        "c5034cf4cc8458612913f3074b7982ed1bb0a4fc2bdb469e61d04436",
+        "be30b6efdbc0ebd80314906ab0f1df971c597fecca6abb4b98be807d",
+        "d17a17939616e8c84cd80cc51d036cadfce7e67fe9ad00d6aaa9b86f",
+        "fb04e22a9196ac9ac1ca806394b291e7e46637aa6b66d697ceb4d24f"
+        ]
+      },
+      {
+        "logoUrl": "https://raw.githubusercontent.com/AlexDochioiu/nft_craze_gallery_data/main/logos/CardanoRx.png",
```

Comment:
The name of the file here is wrong. The actual file name is `CardanoRxThumb.jpg`